Improves docs SEO (#1686)
iotaledger/goshimmer: documentation/docs/apis/ledgerstate.md

---

> The ledgerstate API provides endpoints to retrieve address details, unspent outputs for an address, get branch details, and list child branches amongst others.

Author: ```suggestion 
description: The ledgerstate API provides endpoints to retrieve address details, unspent outputs for an address, get branch details, and list child branches amongst others.
```

> description: The ledgerstate API provides endpoints to retrieve address details, unspent outputs for an address,get branch details , a list of child branches amongst others.  

Author: ```suggestion
The ledgerstate API provides endpoints to retrieve address details, unspent outputs for an address, get branch details, and list of child branches amongst others.
```
